bench/bench-bytestring.cabal

```diff
@@ -153,7 +157,10 @@ benchmark bench-builder-boundscheck
                     gauge
   c-sources:        ../cbits/fpstring.c
                     ../cbits/itoa.c
+                    ../cbits/ftoa.c
+                    ../cbits/dtoa.c
   include-dirs:     ../include
+                    ../cbits
```

Comment:
I think the same change needs to be applied to `bench-builder-csv` and `bench-strict-indices below`.

Reply:
@Lumaere Ping. I think you need to update `include-dirs` for `bench-builder-csv` and `bench-strict-indices` to make CI green.